### PR TITLE
bump azure-core & .gitignore emacs backup files

### PR DESCRIPTION
**Pull request by AOrps**
- Had trouble building new triage-portal so needed to switch azure-core version
- Added emacs backups to the .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -759,3 +759,6 @@ $RECYCLE.BIN/
 # .pnp.*
 
 # End of https://www.toptal.com/developers/gitignore/api/node,yarn,intellij,pycharm,webstorm,python,django,macos,windows,linux
+
+# Emacs backups
+*~
```
Comment (Cyber-JiuJiteria) [inline .gitignore:764]: Why was this added?

Comment (AOrps) [reply .gitignore:764]: Added to prevent emacs backup files that are generated automatically from edits to a file. 
- If I were working on a `src/schema.py` file and using emacs, it would generate a `src/schema.py~` on auto-save to prevent data loss.

Comment (Cyber-JiuJiteria) [reply .gitignore:764]: ooh okay. I visually missed the '.' before the ~. Thanks. This would avoid any temp files to be added as well.

**Review by Cyber-JiuJiteria — APPROVED** [review]
